src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -10,6 +10,55 @@
+            { AppInstaller::Manifest::ManifestError::BothAllowedAndExcludedMarketsDefined, "Both AllowedMarkets and ExcludedMarkets defined." },
+            { AppInstaller::Manifest::ManifestError::DuplicateReturnCodeEntry, "Duplicate installer return code found." },
+            { AppInstaller::Manifest::ManifestError::FieldRequireVerifiedPublisher, "Field usage requires verified publishers." },
+            { AppInstaller::Manifest::ManifestError::SingleManifestPackageHasDependencies, "Package has a single manifest and is a dependency of other manifests." },
+            { AppInstaller::Manifest::ManifestError::MultiManifestPackageHasDependencies, "Deleting the manifest will be break the following dependencies." },
+            { AppInstaller::Manifest::ManifestError::MissingManifestDependenciesNode, "Dependency not found: " },
+            { AppInstaller::Manifest::ManifestError::NoSuitableMinVersion,"No Suitable Minimum Version : " },
+            { AppInstaller::Manifest::ManifestError::FoundLoop, "Loop found." },
+            { AppInstaller::Manifest::ManifestError::ExceededAppsAndFeaturesEntryLimit, "Only zero or one entry for Apps and Features may be specified for InstallerType portable." },
+            { AppInstaller::Manifest::ManifestError::ExceededCommandsLimit, "Only zero or one value for Commands may be specified for InstallerType portable." },
+            { AppInstaller::Manifest::ManifestError::ScopeNotSupported, "Scope is not supported for InstallerType portable." },
+            { AppInstaller::Manifest::ManifestError::InstallerMsixInconsistencies, "Inconsistent value in the manifest." },
+            { AppInstaller::Manifest::ManifestError::OptionalFieldMissing, "Optional field missing." },
+            { AppInstaller::Manifest::ManifestError::InstallerFailedToProcess, "Failed to process installer." },
+            { AppInstaller::Manifest::ManifestError::NoSupportedPlatforms, "No supported platforms." },
+            { AppInstaller::Manifest::ManifestError::ApproximateVersionNotAllowed, "Approximate version not allowed." },
+            { AppInstaller::Manifest::ManifestError::ArpVersionOverlapWithIndex, "DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index.Existing DisplayVersion range in index : " },
+            { AppInstaller::Manifest::ManifestError::ArpVersionValidationInternalError, "Internal error while validating DisplayVersion against index." },
+            { AppInstaller::Manifest::ManifestError::ExceededNestedInstallerFilesLimit, "Only one entry for NestedInstallerFiles can be specified for non-portable InstallerTypes." },
+            { AppInstaller::Manifest::ManifestError::RelativeFilePathEscapesDirectory, "Relative file path must not point to a location outside of archive directory" },
+            { AppInstaller::Manifest::ManifestError::ArpValidationError, "Arp Validation Error" },

```

Comment:
```suggestion
            { AppInstaller::Manifest::ManifestError::ArpValidationError, "ARP Validation Error" },

```

Reply:
Ditto @JohnMcPMS's comment https://github.com/microsoft/winget-cli/pull/2410#issuecomment-1204531101. This change is simply about surfacing specific errors. A look at some of the comments indicates that some more changes could be made in the unit tests, way beyond the scope of the change. 

This might have the ripple of effect of demanding more changes to tests/code that rely on this externally in turn blowing up the scope of externally changes. I'm a bit hesitant to change these messages.